### PR TITLE
Add a link for Package Versions usage documentation

### PR DESCRIPTION
**Pull request by Soullivaneuh**
Could be nice to have it on quick info homepage.

### CODE REVIEW

File: src/Packagist/WebBundle/Resources/views/Web/index.html.twig [hunk 1]
```diff
@@ -15,6 +15,8 @@
     }
 }</code></pre>
 
+            <p>For more information about packages versions usage, see <a href="https://getcomposer.org/doc/01-basic-usage.md#package-versions">composer documentation</a>.</p>
```
Comment (stof) [inline src/Packagist/WebBundle/Resources/views/Web/index.html.twig]: missing ``the``

Comment (Soullivaneuh) [inline src/Packagist/WebBundle/Resources/views/Web/index.html.twig]: Do you mean `the composer documentation`?

Comment (stof) [inline src/Packagist/WebBundle/Resources/views/Web/index.html.twig]: yes, so that it reads like an English sentence.

**Comment by Soullivaneuh** [comment]
Commits squashed after fix. Ready to merge.

**Pull request merged**